tests/pipelines/kit-cluster/run-pipeline.template.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: tekton.dev/v1beta1
```

Comment:
i don't see value in this file too given it's referring to the pipeline that i left a comment about.

Reply:
This will be an example to show how Cx should setup their run.  Especially which service account to be used and what node selector to be used.